Modules/MUON/Common/src/TracksTask.cxx

```diff
@@ -144,32 +158,34 @@ void TracksTask::monitorData(o2::framework::ProcessingContext& ctx)
 {
   ILOG(Info, Devel) << "Debug: MonitorData" << ENDM;
 
-  mRecoCont.collectData(ctx, *mDataRequest.get());
-
-  ILOG(Info, Devel) << "Debug: Collected data" << ENDM;
-
   if (!assertInputs(ctx)) {
     return;
   }
 
+  mRecoCont.collectData(ctx, *mDataRequest.get());
+
+  ILOG(Info, Devel) << "Debug: Collected data" << ENDM;
+
   ILOG(Info, Devel) << "Debug: Asserted inputs" << ENDM;
 
   auto tracks = mRecoCont.getMCHTracks();
   auto rofs = mRecoCont.getMCHTracksROFRecords();
   auto clusters = mRecoCont.getMCHTrackClusters();
   auto digits = ctx.inputs().get<gsl::span<o2::mch::Digit>>("mchtrackdigits");
 
+  ctx.inputs().get<TGeoManager*>("mchgeo");
+
   if (mSrc[GID::MCH] == 1) {
     ILOG(Info, Devel) << "Debug: MCH requested" << ENDM;
     if (true || mRecoCont.isTrackSourceLoaded(GID::MCH)) {
-      ILOG(Info, Devel) << "Debug: MCH source loaded" << ENDM;
```

Comment:
```
if (true || mRecoCont.isTrackSourceLoaded(GID::MCH))
```
the `true` in the `if` statement makes the `isTrackSourceLoaded()` check useless. I suspect this is a leftover of some debugging code that we forgot to cleanup.

Reply:
removed